src/models/Class.js

```diff
@@ -14,12 +14,13 @@ const classSchema = new Schema({
     required: true,
   },
   trainer: {
-    type: Schema.Types.ObjectId,
+    type: [Schema.Types.ObjectId],
+    ref: 'trainer',
     required: true,
-
   },
   activity: {
-    type: Schema.Types.ObjectId,
+    type: [Schema.Types.ObjectId],
```

Comment:
It does not need to be in brackets. Ok for trainer which is an array, but activity is a single identifier.